.github/workflows/application-health-checker.yml

```diff
@@ -30,13 +30,16 @@ jobs:
       - name: Build Project # Build with npm
         run: npm run build
 
+      # Performs the project analysis using SonarQube Scanner.
       - uses: sonarsource/sonarqube-scan-action@v3
         env:
-          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+          SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }} 
           SONAR_HOST_URL: ${{ secrets.SONAR_HOST_URL }}
+
       # If you wish to fail your job when the Quality Gate is red, uncomment the
       # following lines. This would typically be used to fail a deployment.
```

Comment:
```suggestion
```

Reply:
Bu kısmı kesin olarak silelim mi ama daha sonra Quality Gate ayarlarını değiştirmek için kalması daha uygun olmaz mı?

Reply:
Landing tarafında Buğra'nın bir youmu vardı orada söylediği gibi yapabiliriz